sdk/test/logs/logger_sdk_test.cc

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "opentelemetry/sdk/logs/logger.h"
+
+#include <gtest/gtest.h>
+
+using namespace opentelemetry::sdk::logs;
+
+class DummyProcessor : public LogProcessor
+{
+  void OnReceive(std::unique_ptr<opentelemetry::logs::LogRecord> &&record) noexcept {}
+  void ForceFlush(std::chrono::microseconds timeout = std::chrono::microseconds(0)) noexcept {}
+  void Shutdown(std::chrono::microseconds timeout = std::chrono::microseconds(0)) noexcept {}
+};
+
```

Comment:
Would there be logger test cases with `DummyProcessor` ( in current or as part of separate PR). 
Would be good to have below tests:
```
logger.log(r)
logger.log(severity, message, time)
std::map attr= {{"k1", "v1"}, {"k2", "v2"}};
logger.log(severity, name, attr);
logger.log(severity, name, {{"k1", "v1"}, {"k2", "v2"}}};
```
The last one mayn't work, as it may need logger api change ( to support initialization list), but worth the change maybe in another PR.
```

Reply:
Yes good catch - in my opinion these would be better as logging API tests? To follow [traces](https://github.com/open-telemetry/opentelemetry-cpp/blob/master/api/test/trace/noop_test.cc) which does these in the API for example.

Reply:
Right, these would make sense to have them in logging API.